tests/fixtures/noqa.py

```diff
@@ -573,14 +573,21 @@ def literal_none_return_func() -> Literal[None]:  # noqa: WPS701
     """Literal[None]"""
 
 
-def nested_annotation_func(arg: Literal[Literal[1, 2], 3]): # noqa: WPS702
+def nested_annotation_func(arg: Literal[Literal[1, 2], 3]):  # noqa: WPS702
     """Literal[Literal[1, 2], 3]"""
 
 
-def nested_annotation_return_func() -> Union[str, Union[int, float]]: # noqa: WPS702
+def nested_annotation_return_func() -> Union[str, Union[int, float]]:  # noqa: WPS702
     """Union[str, Union[int, float]]"""
 
 
+def optional_union_annotation_func(arg: Optional[Union[str, int]]):  # noqa: WPS704
+    """Optional[Union[str, int]]"""
+
+
+def optional_union_annotation_return_func() -> Optional[Union[str, int]]:  # noqa: WPS704
```

Comment:
One example is enough.

Reply:
@sobolevn why we don't want to check return annotations here? Is it because they are checked in ``test_flat_types.py``? And it's actually checked here for ``Literal[None]`` annotation.

Reply:
Yes, you are correct. We only test here that `noqa: ` comments work. That's it.
Other checks are in `test_flat_types.py`

You can delete this incorrect example with two `Literal` as well. I have just missed it during the review process. Now it causes confusion